tools/docker-compose/compose.yaml

```diff
@@ -14,12 +14,12 @@ services:
       - $PWD/ari/kb:/etc/ari/kb:Z
       - $PWD/tools/scripts:/etc/wisdom/scripts:Z
     ports:
-      - "8000:8000"
+      - "7080:8000"
     expose:
-      - "8000"
+      - "7080"
       - "8001-8011" # django-prometheus
     environment:
-      - DJANGO_SETTINGS_MODULE=ansible_ai_connect.main.settings.development
+      - DJANGO_SETTINGS_MODULE=ansible_ai_connect.main.settings.aap
```

Comment:
Can you please paste the content of this file somewhere?

I need to make the AAP Operator add the env vars to Lightspeed's k8s CR so they're added automagically.

Reply:
responded offline